app/src/main/java/ch/sdp/vibester/helper/GameManager.kt

```diff
@@ -70,6 +66,7 @@ class GameManager : Serializable {
     /**
      * Get a current playing song
      */
+    @JvmName("getCurrentSong1")
```

Comment:
It's Android Studio's suggestion, otherwise I got compile error saying "Platform declaration clash: The following declarations have the same JVM signature (getCurrentSong()Lch/sdp/vibester/model/Song;): 
public open fun <get-currentSong>(): Song defined in ch.sdp.vibester.helper.GameManager
public final fun getCurrentSong(): Song defined in ch.sdp.vibester.helper.GameManage"